### PR TITLE
Update ansible example

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: ansible-linux/Gemfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  gem 'kitchen-vagrant'
+  gem 'kitchen-inspec'
+  gem 'kitchen-sharedtests', '~> 0.2.0'
+  gem 'kitchen-sync'
+  gem 'kitchen-transport-rsync'
+  gem 'kitchen-docker'
+end
+
+group :tools do
+  gem 'github_changelog_generator', '~> 1'
```
Comment (chris-rock) [inline ansible-linux/Gemfile]: I think we do not need the changelog generator here

File: ansible-linux/Gemfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+source 'https://rubygems.org'
+
+group :integration do
+  gem 'test-kitchen', '~> 1.0'
+  gem 'kitchen-ansible'
+  gem 'kitchen-vagrant'
+  gem 'kitchen-inspec'
+  gem 'kitchen-sharedtests', '~> 0.2.0'
```
Comment (chris-rock) [inline ansible-linux/Gemfile]: kitchen-sharedtests is not needed anymore

File: ansible-linux/Gemfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  gem 'kitchen-sync'
+  gem 'kitchen-transport-rsync'
+  gem 'kitchen-docker'
```
Comment (chris-rock) [inline ansible-linux/Gemfile]: we are not using docker here, right?

File: ansible-linux/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+Verify with inspec tests
+
+```
+bundle exec kitchen verify
+```
+
+
+## Usage with Vagrant
```
Comment (chris-rock) [inline ansible-linux/README.md]: It is very nice to see multiple examples

**Comment by chris-rock** [comment]
Fixes #12

**Comment by rndmh3ro** [comment]
I updated the whole Gemfile and threw out every unnecessary gem, thanks @chris-rock !

**Comment by chris-rock** [comment]
Perfect @rndmh3ro

**Pull request merged**